server/nuxeo-nxr-server/src/main/resources/templates/common-base/nxserver/config/cors-config.xml.nxftl

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.ecm.platform.ui.web.cors.config">
```

Comment:
It will be very important to update the CORS documentation as soon as this is merged, in order to explain the new properties, and to stress that if people provide their own XML contribution they will now have to add `<require>org.nuxeo.ecm.platform.ui.web.cors.config</require>` as well.